public_html/admin/language/english/extension/extensions.xml

```diff
@@ -397,6 +397,10 @@
 		<key>text_license_expires</key>
 		<value><![CDATA[Support Till]]></value>
 	</definition>
+	<definition>
+		<key>text_license_expired</key>
+		<value><![CDATA[Extension support has expired or not connected to marketplace]]></value>
```

Comment:
Text correction: Extension support has expired or store is not connected to the marketplace